testing/functional/tests/scheduler/dragAndDrop/presets/widget.setup.ts

```diff
@@ -0,0 +1,22 @@
+import SchedulerTestHelper from '../../../../helpers/scheduler.test.helper';
+import { createWidget, getContainerFileUrl } from '../../../../helpers/testHelper';
+
+import { schedulerDataSource, schedulerResources } from './widget.data';
+
+export const scheduler = new SchedulerTestHelper("#container");
+
+export const createScheduler = async (mode) => {
+    await createWidget("dxScheduler", {
+        dataSource: schedulerDataSource,
+        resources: [{
+            fieldExpr: "resourceId",
+            dataSource: schedulerResources,
+            label: "Color"
+        }],
+        views: [mode],
+        currentView: mode,
+        currentDate: new Date(2018, 3, 20),
+        startDayHour: 9,
+        useColorAsDefault: true
```

Comment:
Why you use this parameter here? Read https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/resources/#useColorAsDefault

Reply:
Yes. It's not needed.